.eleventy.js

```diff
@@ -37,9 +37,16 @@ module.exports = function(eleventyConfig) {
     return collection.getFilteredByGlob(["pages/*md", "pages/*liquid"]);
   });
 
-  // limit filter
-  eleventyConfig.addFilter("limit", function(array, limit) {
-    return array.slice(0, limit);
+  eleventyConfig.addFilter("getBoard", function (arr) {
+    return arr.filter((item) => item.group === "Board");
+  });
+
+  eleventyConfig.addFilter("getAdvisory", function (arr) {
+    return arr.filter((item) => item.group === "Advisory");
+  });
+
+  eleventyConfig.addFilter("getVolunteers", function (arr) {
+    return arr.filter((item) => item.group === "Volunteer");
```

Comment:
This is not ideal, but the combination of liquid + nunjucks means we are limited by what we can use for filtering/sorting.  I was not able to get `where` style liquid filters to work, and I was not able to define a filter that would properly pass in parameters for any eleventy-defined filters.

But this hard-coded filter worked.